plone/namedfile/utils/jpeg_utils.py

```diff
@@ -29,17 +28,18 @@ def process_jpeg(data):
                 if (ord(b) >= 0xC0 and ord(b) <= 0xC3):
                     jpeg.read(3)
                     h, w = struct.unpack('>HH', jpeg.read(4))
+                    content_type = 'image/jpeg'
```

Comment:
@thet could I ask why you have moved this statement over here, because it is not consitent with the JPEG specification. 

Actually the two first byte '\377\330' indicate that it is a JPEG image. 

Also the way this we try to look up the width and hight of an Image might conflict a bit more. As the approach is to first look at only 1024 byte of the potential image, where this information may not be contained, but the content_type is always included in the first bytes (see UNIX magic bytes)